pages/reference/OS/updates/self-service.md

```diff
@@ -37,7 +37,7 @@ For device types and {{ $names.os.lower }} versions that are not yet supported,
 
 To run an update for an individual device, navigate to that device's *Actions* tab, click *Update {{ $names.os.upper }}*, and select the version of {{ $names.os.lower }} you would like to update to:
 
-<img src="/img/common/updates/update-os.png" width="80%">
+<img src="/img/common/updates/update-os.png" width="70%">
```

Comment:
is this change accidental, or somehow related to ESR?

Reply:
Good catch and it's intended. I squashed the commits already so this isn't clear. There is a link to the self-service update docs that showed a very dated host OS image update which I refreshed. I also used the existing filename, which in retrospect is obviously not the best idea as no easy way of telling what has been updated other than the proportions of the image changed slightly.